app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useState, useEffect } from "react";
+import {
+  Checkbox,
+  IconButton,
+  InputBase,
+  ButtonBase,
+  Popper,
+  fade,
+  makeStyles,
+  createStyles,
+} from "@material-ui/core";
+import Autocomplete, {
+  AutocompleteCloseReason,
+} from "@material-ui/lab/Autocomplete";
+import { CloseIcon, ExpandMore } from "../../components/Icons";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    popper: {
+      border: "1px solid rgba(27,31,35,.15)",
+      boxShadow: "0 3px 12px rgba(27,31,35,.15)",
+      borderRadius: 3,
+      zIndex: 1,
+      color: "#586069",
+      backgroundColor: "#f6f8fa",
```

Comment:
If it's relevant, do you want to put these in the palette in theme? Or maybe we should save that for a later PR.

Reply:
Yup, changed most values to use theme. Left some others just to match the current default input field.